source/localizable/routing/redirection.md

```diff
@@ -27,23 +37,27 @@ Router.map(function() {
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  beforeModel() {
-    this.transitionTo('posts');
+  beforeModel(/* transition */) {
+    this.transitionTo('posts'); // Implicitly aborts the on-going transition.
   }
 });
 ```
 
+`beforeModel()` receives the current transition as an argument, which we can store and retry later.
+This allows us to return the user back to the original route.
+For example, we might redirect a user to the login page when they try to edit their profile, and immediately redirect
```

Comment:
I added a link to the Guides on [Storing and Retrying Transitions](https://guides.emberjs.com/v2.9.0/routing/preventing-and-retrying-transitions/#toc_storing-and-retrying-a-transition). If that's not enough we could use diagrams.